NinchatSDKSwift/Implementations/Data Source - Delegate/Questionnaire/QuestionnaireDataSourceDelegate.swift

```diff
@@ -246,4 +239,20 @@ extension QuestionnaireDataSourceDelegate {
             ]
             """
     }
+
+    internal func registeredLogic() -> String {
```

Comment:
Interesting approach. Why _registered can't just be displayed directly? If someone adds an element named "registered-logic" in their questionnaire, what happens then?

Reply:
To show the `_registered` element, we need to have either a `redirect` or `logic` blocks, or to be ensure that it is placed in the right place that we don't need any of those blocks. But according to the issue, the element can be placed anywhere.